service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreProxy.java

```diff
@@ -55,6 +59,28 @@ Licensed to the Apache Software Foundation (ASF) under one
 
     private IMetaStoreClient client = null;
 
+    @Value("${hive.krb5conf.path}")
+    private String hiveKrb5confPath;
+
+    @Value("${hive.keytab.path}")
```

Comment:
ok, I accept your suggestion

Reply:
> Since kerberos doesnt apply to just hive but hadoop services in general, the "hive." prefix can be removed from all these configs.
emmm, kerberos have many keytabs for different user,like hive,hdfs,livy and so on.
i can see many keytab files name like hive.keytab,hdfs.keytab...
how about keep the prefix?